### PR TITLE
Prettier and ESLint import rules

### PR DESCRIPTION
**Pull request by AG-Guardian**

### CODE REVIEW

**Comment by AG-Guardian** [comment]
Feel free to reject this if you prefer less strict rules. Prettier is pretty much just using defaults. ESLint rules will group imports by external packages vs source packages, and sort accordingly. 

No code is changed here, only formatting of files.

File: eslint.config.js [hunk 1]
```diff
@@ -1,28 +1,45 @@
...
     languageOptions: {
       ecmaVersion: 2020,
       globals: globals.browser,
     },
     plugins: {
-      'react-hooks': reactHooks,
-      'react-refresh': reactRefresh,
+      "react-hooks": reactHooks,
+      "react-refresh": reactRefresh,
+      "simple-import-sort": eslintPluginSimpleImportSort,
```
Comment (scottbenton) [inline eslint.config.js:26]: This is a wonderful change, thank you! Much more organized!

File: eslint.config.js [hunk 2]
```diff
@@ -1,28 +1,45 @@
...
+      "no-relative-import-paths": noRelativeImportPaths,
```
Comment (scottbenton) [inline eslint.config.js]: This one I'm a little less sure of, but I'm not fully against it - I'd like to hear out the reasoning a bit more!

If I'm referencing something in the same directory, isn't it easier (and clearer to read) to say  `import {Button} from './Button'` than `import {Button} from 'components/shared/Button/Button'`?

File: src/components/datasworn/AssetCard/AssetControl.tsx [hunk 3]
```diff
@@ -1,15 +1,16 @@
+import { useCallback } from "react";
 import { Datasworn } from "@datasworn/core";
 import { Box } from "@mui/material";
-// import { Track } from "components/features/Track";
-import { AssetControls } from "./AssetControls";
+
 import { AssetDocument } from "api-calls/assets/_asset.type";
-import { AssetCounterField } from "./fields/AssetCounterField";
-import { AssetClockField } from "./fields/AssetClockField";
```
Comment (scottbenton) [inline src/components/datasworn/AssetCard/AssetControl.tsx]: This for example, is where I feel that relative imports are clearer than absolute.

**Review by scottbenton — COMMENTED** [review]
Great changes overall, just wanted to pick your brain on relative vs absolute imports.

File: eslint.config.js (hunk 2 above)
Comment (AG-Guardian) [reply eslint.config.js]: the philosophy is to make developers more aware of the folder structure hopefully keep things organized, but I am actually going to revert this - absolute paths are not working for storybook and I was not super in love with them to begin with

Comment (scottbenton) [reply eslint.config.js]: Sounds good! Thanks

File: .storybook/tsconfig.json
Comment (AG-Guardian) [inline .storybook/tsconfig.json:1]: This is just being added to make the IDE happy

**Comment by AG-Guardian** [comment]
Ok - latest revision has reverted the absolute path imports, but everything else is the same. Storybook is working again :)

**Review by scottbenton — APPROVED** [review]

**Pull request merged**